model/event/subscriptions.go

```diff
@@ -238,14 +248,26 @@ func (s *Subscription) String() string {
 	return out
 }
 
-func FindSubscriptionsByOwner(owner string) ([]Subscription, error) {
+func FindSubscriptionsByOwner(owner string, ownerType OwnerType) ([]Subscription, error) {
 	if len(owner) == 0 {
```

Comment:
Validity check on ownerType? This should return bad request in that case.

Reply:
There's a check in the validate function of the route. The function signature here will prevent invalid values